parallaxinc/solo
# Set default block to legal value

Addresses #107 

To test:  
 - Drag Badge set LED block onto workspace.  
 - Make sure it generates `led(0, 1);`
